fitbenchmarking/cli/main.py

```diff
@@ -142,11 +154,17 @@ def run(problem_sets, options_file='', debug=False):
 
         # resets options to original values
         options.reset()
+    if os.path.basename(options.results_dir) == \
+            options.DEFAULT_PLOTTING['results_dir']:
```

Comment:
Somewhat unrelated to this PR, but what is the reason that 'results_dir' is located within

```
    DEFAULT_PLOTTING = \
        {'make_plots': True,
         'colour_scale': [(1.1, "#fef0d9"),
                          (1.33, "#fdcc8a"),
                          (1.75, "#fc8d59"),
                          (3, "#e34a33"),
                          (np.inf, "#b30000")],
         'comparison_mode': 'both',
         'table_type': ['acc', 'runtime', 'compare', 'local_min'],
         'results_dir': 'results'}
```

Does PLOTTING in DEFAULT_PLOTTING refer to anything that fitbenchmarking returns to the user? Maybe discuss this later in todays' friday catch-up